JustSaying.TestingFramework/IntegrationAwsClientFactory.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using Amazon;
+using Amazon.Runtime;
+using Amazon.SimpleNotificationService;
+using Amazon.SQS;
+using JustSaying.AwsTools;
+
+namespace JustSaying.TestingFramework
+{
+    /// <summary>
+    /// AwsCustomClient factory for running integration tests in continuous integration mode.
+    /// 
+    /// If environment variable "ci" is not defined, use default AWS profile name as specified by <code>IntegrationTestConfig.AwsProfileName</code>
+    /// 
+    /// Otherwise, construct AWS Profile using access and secret key by looking at "ci-access_key" and "ci-secret_key" environment variables.
+    /// </summary>
+    public class IntegrationAwsClientFactory : IAwsClientFactory
+    {
+        private readonly AWSCredentials _credentials;
+        private readonly string _ci = "ci";
+        private readonly string _ciAccesskey = "AWS_ACCESS_KEY_ID";
+        private readonly string _ciSecretkey = "AWS_SECRET_KEY";
```

Comment:
@dariusdamalakas You're completely correct :grin: I'd prefer to be consistent with the C# world :+1: